pages/common/mariadb.md

```diff
@@ -0,0 +1,36 @@
+# mariadb
+
+> The mariadb Command-Line Client tool.
+> More information: <https://mariadb.com/kb/en/mariadb-command-line-client/>.
+
+- Connect to a specific MariaDB database:
+
+`mariadb {{db_name}}`
+
+- Connect to a specific MariaDB database {{db_name}} using {{user_name}} and {{your_password}}:
+
+`mariadb --user={{user_name}} --password={{your_password}} {{db_name}}`
+
+- Display help and exit:
+
+`mariadb -?, --help`
+
+- Show warnings after every statement in interactive and batch mode:
+
+`mariadb --show-warning`
+
+- Be more silent. Can be used multiple times to produce less output:
```

Comment:
```suggestion
- Display less verbose outputs:
```

Reply:
```suggestion
- Display less verbose outputs (can be used multiple times to produce less output):
```
I would keep the additional note about using it multiple times to clarify the command example.